client/blocks/app-banner/utils.js

```diff
@@ -61,6 +63,13 @@ export function getCurrentSection( currentSection, isNotesOpen ) {
 	return null;
 }
 
+export function getCurrentPathFragment( state ) {
+	const route = getCurrentRoute( state );
+
+	// Strip off the leading forward slash for use as a fragment
```

Comment:
an example of the input data could make this comment much more useful to an unfamiliar reader - what does `getCurrentRoute()` return?

```js
// strip off the leading forward-slash for use as a fragment
// e.g. /settings/mysite.wordpress.com -> settings/mysite.wordpress.com
```

also, why are we stripping the slash?

Reply:
Definitely agree with the clearer example. If I'm completely honest, there's reason for stripping the leading slash other than I thought it looked nicer. I understand that's not a great reason, and perhaps it'd be best to just leave it in there :)

Reply:
Seems like a bit of overkill to add this trivial util function before we have a second instance, it's easy enough to strip a leading slash in the deep link function as well.

Reply:
This is technically a selector too, which would normally live in `state/selectors` I don't think we need this though. It's enough to modify in `mapStateToProps` or in the deep link function directly.